uplead (#1307)
n8n-io/n8n-docs: docs/integrations/builtin/app-nodes/n8n-nodes-base.uplead.md

# UpLead

> The UpLead node allows you to automate work in UpLead, and integrate UpLead with other applications. n8n has built-in support for a wide range of UpLead features, including getting company information. 

Reviewer: ```suggestion
The UpLead node allows you to automate work in UpLead, and integrate UpLead with other applications. n8n supports several UpLead operations, including getting company information. 
```
